torch/onnx/__init__.py

```diff
@@ -39,8 +39,10 @@ def set_training(model, mode):
         if old_mode != mode:
             model.train(old_mode)
 
+# The global variable indicates whether aten mode is on
+aten=False
```

Comment:
Make this private with an underscore.

Reply:
I don't want to be blocking here, but why do we need this global? It would be much clearer to make `_run_symbolic_function` a closure that captures `aten` value local to the `export` call and then we pass this pointer as a `PyObject*` to the C backend. This would let us not have this code that looks it up from C (you just get it from a pointer), and we wouldn't have to use globals (ugh)

Reply:
Okay, let me remove this global.